Formula/cln.rb

```diff
@@ -1,7 +1,7 @@
 class Cln < Formula
   desc "Class Library for Numbers"
   homepage "https://www.ginac.de/CLN/"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
```

Comment:
This should be `GPL-2.0-or-newer`. See: https://www.ginac.de/CLN/cln.git/?p=cln.git;a=blob;f=COPYING;h=20774a1d47c8194ae5c09b589094735c0e977772;hb=HEAD#l300

```suggestion
  license "GPL-2.0-or-newer"
```

Reply:
Yeah, I omitted it.
Thank you~